dist/index.d.ts

```diff
@@ -49,7 +49,22 @@ export type ErrorMessage = Partial<MessageBase> & {
 export type AckMessage = {
 	id: string;
 };
-export type Message = MempoolMessage | BlockMessage | ErrorMessage | AckMessage;
+export type Stats = {
+	erc20: number;
+	erc721: number;
+	erc777: number;
+	estimatedMaxFeePerGas: number[];
```

Comment:
Might need to run build and commit the dist again? I plan to remove the dist folder from the repo, but need to get bun installed in to our build pipeling.